Sources/Components/Feedback/FeedbackView.swift

```diff
@@ -145,16 +145,12 @@ public class FeedbackView: UIView {
     }
 
     @objc private func positiveButtonTapped() {
-        if #available(iOS 10.0, *) {
-            UIImpactFeedbackGenerator(style: .light).impactOccurred()
-        }
+        UIImpactFeedbackGenerator(style: .light).impactOccurred()
         delegate?.feedbackView(self, didSelectButtonOfType: .positive, forState: state)
     }
 
     @objc private func negativeButtonTapped() {
-        if #available(iOS 10.0, *) {
-            UIImpactFeedbackGenerator(style: .light).impactOccurred()
-        }
+        UIImpactFeedbackGenerator(style: .light).impactOccurred()
```

Comment:
Maybe we could pull `UIImpactFeedbackGenerator` into an instance var now? Not sure if it has any impact (pun intended) on performance, but it makes sense to not create it each time we need it.

Reply:
Yeah, good idea! Should keep this PR as is.